.rubocop_local.yml

```diff
@@ -1,3 +1,5 @@
 Style/GlobalVars:
   AllowedVariables:
     - $fog_log
+Lint/RescueException:
+  Enabled: false
```

Comment:
We should probably leave this up to the core manageiq-style (discussion ongoing over in https://github.com/ManageIQ/manageiq-style/pull/15) rather than override it here

Reply:
Ok, removed it.